assets/js/components/Popup.js

```diff
@@ -3,8 +3,10 @@ const utils = require('../libs/elife-utils')();
 module.exports = class Popup {
 
   constructor($elm, _window = window, doc = document) {
-    if (!$elm.hash || $elm.host !== _window.location.host || !$elm.hash.match(/^#[a-z]/i)) {
-      return;
+    if (!$elm.dataset.popupSelf) {
```

Comment:
Introducing this feels a bit like a hack, but the element isn't an `<a>` and it doesn't have an ID.